test/test_optim.py

```diff
@@ -798,6 +798,42 @@ def test_lambda_lr(self):
                              lr_lambda=[lambda x1: 0.9 ** x1, lambda x2: 0.8 ** x2])
         self._test(scheduler, targets, epochs)
 
+    def test_sgdr_lr1(self):
+        iters = 100
+        eta_min = 1e-10
+        T_mults = [1, 2]
```

Comment:
Good, but it made me think about `T_mult >= 1 (e.g. 2.5) and T_mult < 1 (e.g. 0.5)`. I think the latter is not practical, so I modified the range of `T_mult`, if < 1, raise `ValueError`.
What do you think?

By the way, because of `T_mults` could be float point number, so the test cases were slightly changed to test the case like `T_i = 62.5`

Reply:
I agree with you on `T_mult >= 1`, as the original paper says:

> we suggest an option to start with an initially small `T_i` and **increase** it by a factor of `T_mult` at every restart